src/nedap/utils/test/impl.cljc

```diff
@@ -83,3 +83,12 @@
                    (str (pr-str to-change#) " does not match expected from: " (pr-str from#)))))
        ~@bodies
        (~is (meta= [~to-change ~to])))))
+
+#?(:clj
+    (defmacro when-matcher-combinators
```

Comment:
Can't we just put `matcher-combinators` in provided deps and ditch this macro? I'm pretty sure we did that in our consul component.

Would help cursive out, this macro prevents it from 'recognising' the `in-any-order` symbol.

Reply:
Right now `utils.test` usable without `matcher-combinators`-dep. Making it a `provided` dependency; `utils.test` cannot be loaded without it

Reply:
Hey, I tried this, because I was pretty sure you can use libraries without providing all provided deps. And it turns out you can.

So I removed the macro around `in-any-order`, added m-c as a provided dep, and installed it.

I created another project depending on `utils.test`, but without matcher combinators:

<img width="542" alt="Screenshot 2020-06-15 at 12 56 17" src="https://user-images.githubusercontent.com/37534243/84649946-ab317b00-af07-11ea-9242-24fc043ba4a7.png">

And I can open a repl in there and require `nedap.utils.test.api` (and obviously not `nedap.utils.test.matchers`):

<img width="611" alt="Screenshot 2020-06-15 at 12 56 30" src="https://user-images.githubusercontent.com/37534243/84649969-b4bae300-af07-11ea-9643-e4560d494238.png">

Am I missing something here?

Reply:
🤔  Now I _am_ intrigued though. Maybe this becomes an issue when (aot) compiling? Not sure. Will think about this. Rather have a simpler solution in this  case (e.g. just use `provided` and an actual error upon load)